src/ucs/Makefile.am

```diff
@@ -106,6 +106,7 @@ noinst_HEADERS = \
 	stats/stats.h \
 	sys/checker.h \
 	sys/compiler.h \
+	sys/lib_util.h \
```

Comment:
maybe path.{h,c} (as in python's os.path)?
or self.{h/c} / just lib.{h,c}
(util word is weird because all ucs is utils..)